### PR TITLE
Rpc endpoints

### PR DESCRIPTION
**Pull request by 0o-de-lally**
Refactor how seed peers get created so that:

- [x] nodes are not filtered from the playlist if they are offline at time of refreshing peers (allow `txs` to pick a peer). Peers lists were coming up empty.
- [x] change format of the "playlist" to include some notes on host.
- [x] allow for a "custom" type of network, where the user can define a different URL of a "playlist"
- [x] make the upstream_peers list randomized when it gets populated.
- [x] change location in repo for template URLs
- [x] no hardcoded URLs in Rust code.

### CODE REVIEW

File: seed_peers/fullnode_seed_playlist.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "note": "daniyal",
+      "url": "http://135.181.118.28:8080"
+    },
+    {
+      "note": "nazario",
+      "url": "http://0l.fullnode.gnazar.io"
+    }
+  ]
+}
```
Comment (dboreham) [inline seed_peers/fullnode_seed_playlist.json]: Missing newline

File: seed_peers/fullnode_seed_playlist_testnet.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "nodes": [
+    {
+      "note": "Rex - Alice",
+      "url": "http://64.225.2.108:8080"
+    }
+  ]
+}
```
Comment (dboreham) [inline seed_peers/fullnode_seed_playlist_testnet.json]: Missing newline

File: public/fullnodes.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
       "http://165.232.136.149:8080",
```
Comment (dboreham) [inline public/fullnodes.json]: Nit, but it would be nice if the list only appears once.

**Review by dboreham — APPROVED** [review]

File: public/fullnodes.json (hunk 3 above)
Comment (0o-de-lally) [reply public/fullnodes.json]: yes, this is deprecated

File: seed_peers/fullnode_seed_playlist.json (hunk 1 above)
Comment (0o-de-lally) [reply seed_peers/fullnode_seed_playlist.json]: watching those newlines like a hawk

**Pull request merged**